test/common/concepts_common.h

```diff
@@ -235,20 +235,51 @@ template <typename T, typename F = T> using WithFeederOperatorRoundBracketsNonCo
 template <typename T, typename F = T> using WithFeederWrongFirstInputOperatorRoundBrackets = ParallelForEachFeederBody<T, F, /*() = */State::incorrect_first_input>;
 template <typename T, typename F = T> using WithFeederWrongSecondInputOperatorRoundBrackets = ParallelForEachFeederBody<T, F, /*() = */State::incorrect_second_input>;
 } // namespace parallel_for_each_body
-namespace container_based_sequence {
+namespace parallel_sort_value {
+template<bool MovableV, bool MoveAssignableV, bool ComparableV>
+struct ParallelSortValue
+{
+    ParallelSortValue(ParallelSortValue&&) requires MovableV {}
+    ParallelSortValue& operator=(ParallelSortValue&&) requires MoveAssignableV {return *this;}
+
+    friend bool operator<(const ParallelSortValue&, const ParallelSortValue&) requires ComparableV { return true; }
+};
+
+template<bool MovableV, bool MoveAssignableV, bool ComparableV>
+void swap(ParallelSortValue<MovableV, MoveAssignableV, ComparableV>&,
+          ParallelSortValue<MovableV, MoveAssignableV, ComparableV>&)
+{}
 
-using iterator = int*;
+using NonMovableValue = ParallelSortValue</*MovableV = */false, /*MoveAssignableV = */true, /*ComparableV = */true>;
```

Comment:
It seems like we also need `Correct` value:
```suggestion
using Correct = ParallelSortValue</*MovableV = */true, /*MoveAssigbaleV = */true, /*ComparableV = */true>;
using NonMovableValue = ParallelSortValue</*MovableV = */false, /*MoveAssignableV = */true, /*ComparableV = */true>;
```